tests/end-to-end/cli/filter/filter-dataprovider-by-phpstorm-regexp.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+phpunit --filter '/PHPUnit\\TestFixture\\DataProviderFilterTest::testFalse with data set \"other false test\"$/' ../../_files/DataProviderFilterTest.php
```

Comment:
usually we put a url to the underlying issue/pr in `--TEST--`.

in this case it would be
```
--TEST--
https://github.com/sebastianbergmann/phpunit/pull/6364
```